src/Pages/UserHint/UserHint.jsx

```diff
@@ -0,0 +1,21 @@
+import { useNavigate } from 'react-router-dom'
+import { JsData } from '../../Backend/Data'
+import { Account } from '../../Components/Other/Account'
+import { LANDING_PAGE_PATH } from '../../Constants/Constants'
+
+const allCompanies = JsData.searchData.companies
+
+export const UserHint = () => {
+  const navigate = useNavigate()
+
+  return (
+    <Account
+      title='Beneath are the companies in the database'
+      onClick={() => navigate(LANDING_PAGE_PATH)}
+      btnTitle='To Frontpage'
+      content={allCompanies.map(({ companyName }) => (
+        <p>{companyName}</p>
```

Comment:
instead of content maybe you could render {children} in Account component, it will give you more flexibility especially if more or longer content is in use